_amenities/klahanie-neighborhood-patrol.md

```diff
@@ -0,0 +1,34 @@
+
+Currently, your KNP is comprised of 23 fully-commissioned Deputies, Detectives and Sergeants. Each of us is employed by the King County Sheriff’s Office.  Perhaps you have noticed patrol cars with the names of other communities driving through your neighborhood?  Those officers are not lost, nor do they live here; instead, they are on patrol protecting you and your family. Each officer has permission to use their patrol vehicles while working in Klahanie. The time spent here is in addition to their regular jobs—this is referred to as “off-duty” work.
+
+<strong>Police, Sheriff or Both?</strong>
+
+The City of Sammamish contracts with the Sheriff’s office to provide deputies to staff the Police Department. These officers are tasked with patrolling the incorporated area of Sammamish, which includes Klahanie.  However, Sammamish is a large geographical area with a rapidly growing population.  For this reason, your Klahanie Homeowner’s Association created the KNP, whose sole responsibility is to provide enhanced police services to ensure the public safety of the Klahanie community.  The KNP have all the legal powers to ticket, arrest or apprehend anyone breaking the law.
+
+While in-service, the KNP Deputies are actively monitoring all emergency and non-emergency radio traffic occurring both inside and outside of Klahanie.  The KNP is staffed daily by three or more Deputies, on staggered shifts, to ensure a cost-effective use of the Klahanie Public Safety budget.  We drive both marked and unmarked police vehicles, and work in uniformed and plain-clothes capacities.  Our staff currently includes five sergeants, four detectives, two SWAT team members, and two K9s, each with a depth of training and experience.  Our daily patrols are usually staffed during morning and early afternoon hours, when burglaries are most likely to occur, as well as late evening and overnight when vehicle prowls are most common.
+
+The enhanced services offered by your KNP include but are not limited to: foot and bike patrols of our 30 miles of trail systems, additional parking and school zone enforcement, and shortened response times to both emergency and non-emergency calls for service.  Further, we work in cooperation with your HOA staff to address any problems that arise, assist with creating block watches, attend community meetings, provide security for many of the events hosted by the Klahanie HOA, and conduct package theft patrols.  Perhaps our most popular service is the Vacation Security Check program in which deputies check your house daily while you’re away.  The form for requesting this service is available through the HOA office and is posted on the Klahanie website.
+
+As residents of Sammamish, and due to the efforts of your HOA, Klahanie residents enjoy unparalleled police staffing found nowhere else in King County.  We are proud to be your Klahanie Neighborhood Patrol—so don’t ever hesitate to call upon us if we can be of service.
```

Comment:
Because this paragraph ends with "don't ever hesitate to call upon us," there should be contact links at the bottom of this page.